src/panels/config/js/trigger/device.tsx

```diff
@@ -41,6 +90,19 @@ export default class DeviceTrigger extends Component<any, any> {
           hass={hass}
           label="Trigger"
         />
+        <paper-time-input
+          label={hass.localize(
+            "ui.panel.config.automation.editor.triggers.type.state.for"
+          )}
+          name="for"
+          hour={hours}
+          min={minutes}
+          sec={seconds}
+          enable-second
+          format={24}
+          onvalue-changed={this.durationChanged}
+          hidden={!showFor}
```

Comment:
This doesn't work, is it a bug in paper-time-input?

Reply:
So `hidden` only works in Polymer if we add the CSS and set it as an attribute. In Preact, you need to conditionally render it:

```tsx
{showFor && <paper-time-input … />}
```

Reply:
Thanks, fixed.